files/zh-cn/web/html/element/u/index.md

```diff
@@ -112,7 +114,7 @@ u.spelling {
 <p>The class read <cite>Moby Dick</cite> in the first term.</p>
 ```
 
-{{EmbedLiveSample("使用 cite 元素", 650, 80)}}
+{{EmbedLiveSample("使用_cite_元素", 650, 80)}}
```

Comment:
please revert this, actually, I prefer the previous one (although both of them are correct). See: https://github.com/orgs/mdn/discussions/448#discussioncomment-6974683

Reply:
got it